pkg/views/build/list/view.go

```diff
@@ -44,34 +36,16 @@ func ListBuilds(buildList []apiclient.Build, apiServerConfig *apiclient.ServerCo
 		data = append(data, row)
 	}
 
-	terminalWidth, _, err := term.GetSize(int(os.Stdout.Fd()))
-	if err != nil {
-		fmt.Println(data)
-		return
-	}
-
-	breakpointWidth := views.GetContainerBreakpointWidth(terminalWidth)
-
-	minWidth := views_util.GetTableMinimumWidth(data)
+	table, success := views_util.GetTableView(data, []string{
```

Comment:
Since, in every file, we have
``` golang
table, success := GetTableView(data, headers)
if !success {
  renderUnstyledList(...)
 return
}

fmt.Println(table)
```

I think this might be cleaner.

```golang
func RenderTableView(data [][]string, headers []string, renderFallback func()) { ... }
---

views_util.RenderTableView(data, headers, func() { renderUnstyledList(...) })

```

(+ we add a comment that says when the renderFallback is triggered.

Reply:
Had to keep `GetTableView` instead of `RenderTableView` because it is needed for the ProfileList and might be useful in the future for rendering the table as a part of a bigger output string